modules/apps/29-fee/types/msgs.go

```diff
@@ -156,20 +151,30 @@ func (fee IdentifiedPacketFee) Validate() error {
 		return sdkerrors.Wrap(err, "failed to convert RefundAddress into sdk.AccAddress")
 	}
 
+	// enforce relayer is nil
+	if fee.Relayers != nil {
+		return ErrRelayersNotNil
+	}
+
+	// validate Fee
+	if err := fee.Fee.Validate(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Validates that each Fee is valid or if all three Fees are empty or zero
```

Comment:
Godoc should start with function name.

```suggestion
// Validate asserts that each Fee is valid and all three Fees are not empty or zero
```